devtools/conda-envs/test_env.yaml

```diff
@@ -24,4 +24,5 @@ dependencies:
   - gromacs =2019
   - lammps
   - mbuild
+  - parmed
   - foyer
```

Comment:
```suggestion
  - foyer
  - foyer >=0.8.0
```

So as to avoid pulling down an older version without the new API points

Reply:
Its available [here](https://anaconda.org/conda-forge/foyer). But, I cannot pull it locally, neither is CI pulling it.

Reply:
Yeah, it's unavailable to `conda` for an hour or two after it shows up on the website. Something about CDN caching that I don't understand - I just know to wait for a bit

Reply:
Okay. Will try again in a bit.